source/algorithms.tex

```diff
@@ -5190,7 +5190,7 @@
 \pnum
 \complexity
 At most
-$\max(\lfloor{\frac{3}{2}} (N-1)\rfloor, 0)$
+$\max(\left\lfloor{\frac{3}{2}} (N-1)\right\rfloor\mathclose{}, 0)$
```

Comment:
Ah nice, using \bigl and \bigr produces the same result without needing the \mathclose, thanks!

Patch amended.